### DISCUSSION TITLE
求救Failed to load resource: net::ERR_CONNECTION_TIMED_OUT || Help Failed to load resource: net::ERR_CONNECTION_TIMED_OUT

### DISCUSSION THREAD

**Discussion by lyay23** [walinejs/waline#2787] [Q&A] [answered]
### Describe the bug | 问题描述

正常部署waline，不过没有买域名，评论时出现错误Failed to load resource: net::ERR_CONNECTION_TIMED_OUT WalineComment.vue:194   Failed to fetch (匿名)	@	WalineComment.vue:194,求大佬帮助

### Website URL | 问题网站

lyay23.github.io

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

Waline is deployed normally, but the domain name is not purchased. An error occurs when commenting: Failed to load resource: net::ERR_CONNECTION_TIMED_OUT WalineComment.vue:194 Failed to fetch (anonymous) @ WalineComment.vue:194, please help me

### Website URL | Problem website

lyay23.github.io

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
我这边看你的网站已经是好的了，应该已经没有问题了？

---
I see that your website is already good, so there should be no problems?

**Comment by lyay23** [comment, marked as answer]
没问题了，绑完域名就好了，谢谢啦



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Austin Lee ***@***.***&gt;
发送时间: 2024年12月17日 08:21
收件人: walinejs/waline ***@***.***&gt;
抄送: 李阳 ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [walinejs/waline] 求救Failed to load resource: net::ERR_CONNECTION_TIMED_OUT || Help Failed to load resource: net::ERR_CONNECTION_TIMED_OUT (Discussion #2787)

---
No problem. Just bind the domain name. Thank you.



Sent from my iPhone


------------------ Original email ------------------
Sender: Austin Lee ***@***.***>
Sent time: December 17, 2024 08:21
Recipient: walinejs/waline ***@***.***>
Cc: Li Yang ***@***.***>, Author ***@***.***>
Topic: Re: [walinejs/waline] Help Failed to load resource: net::ERR_CONNECTION_TIMED_OUT || Help Failed to load resource: net::ERR_CONNECTION_TIMED_OUT (Discussion #2787)
